src/gui/static/src/app/components/pages/force-skywallet-wipe/force-skywallet-wipe.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+import { FormBuilder } from '@angular/forms';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+
+import { ChildHwDialogParams } from '../../layout/hardware-wallet/hw-options-dialog/hw-options-dialog.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163601)
```suggestion
import { ChildHwDialogParams } from "../../layout/hardware-wallet/hw-options-dialog/hw-options-dialog.component";
```